tests/forms/my.applications.spec.js

```diff
@@ -7,64 +7,59 @@ configure({ adapter: new Adapter() });
 let DefaultService = require('../../src/service/default.service');
 
 describe('MyApplications component', ()=> {
-    
+        
```

Comment:
extra space in the empty line.  I wonder why?  Sometimes my IDE does this to me.